### DISCUSSION TITLE
Issue pressing left `⌥` + `3` to get hashtag character (`#`) on mac

### DISCUSSION THREAD

**Discussion by benjaminjellis** [wezterm/wezterm#5116] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I've noticed that when trying to get a hashtag `#`  character in wezterm by pressing left `⌥` + `3` on mac doesn't do anything. Pressing right `⌥` + `3` yields the expected hashtag character. I've checked another terminal (iterm2) to verify that this isn't a hardware / keyboard issue and it works just fine. I'm not sure if this is a bug or if it's a config issue. Any help would be greatly appreciated!

**Comment by voidptrr** [comment]
Getting same issue on this one

**Comment by dagrlx** [comment, marked as answer]
you must activate this setting in your configuration

config.send_composed_key_when_left_alt_is_pressed = true
config.send_composed_key_when_right_alt_is_pressed = true

**Reply by voidptrr** [reply]
Thanks, this works

**Reply by benjaminjellis** [reply]
Thanks!
